config/environments/test.rb

```diff
@@ -31,22 +31,18 @@
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
 
-  ActiveSupport::Deprecation.silenced = true
+  # Randomize the order test cases are executed.
+  config.active_support.test_order = :random
 
-  # For rails-settings-cached conflict
-  config.cache_store = :file_store, 'tmp/cache_test'
+  # Print deprecation notices to the stderr.
```

Comment:
`Raise an error on deprecation notices`, print to stderr is `:stderr`

Reply:
Fixed